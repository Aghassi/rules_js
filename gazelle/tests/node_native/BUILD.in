# gazelle:ts_environment node
