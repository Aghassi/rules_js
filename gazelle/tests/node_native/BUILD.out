load("@npm//@bazel/typescript:index.bzl", "ts_project")

# gazelle:ts_environment node

ts_project(
    name = "node_native",
    srcs = ["main.ts"],
)